.changes/v3.9.0/948-improvements.md

```diff
@@ -0,0 +1,2 @@
+* `vcd_external_network_v2` allows setting DNS fields `dns1`, `dns2` and `dns_suffix` for NSX-T
+  backed entities [GH-948]
```

Comment:
Would it be possible to list the use case in here as well? Or are there more than one?

Reply:
So far there is one known situation - I have added it now.